shared/js/background/utils.es6.js

```diff
@@ -230,6 +236,30 @@ function isSafeListed (url) {
     return false
 }
 
+function isCookieExcluded (url) {
+    const domain = (new URL(url)).host
+    return isDomainCookieExcluded(domain)
+}
+
+function isDomainCookieExcluded (domain) {
+    const excludeList = configStorage.config.features?.cookieProtections.exceptions
```

Comment:
Protections has "cookie" which will be needed still as protections.js lists out the features it expects (do-not-track is being removed). Perhaps combining both `cookie` and `cookieProtections`